content/news/2015/10/2015-10-13-trends-on-tuesday-is-crowdsourcing-the-future-of-mobile-design-and-development-2.md

```diff
@@ -8,7 +8,6 @@ authors:
 topics:
   - crowdsourcing-and-citizen-science
   - mobile
-  - monthly-theme
   - apps-challenges
```

Comment:
```suggestion
  - apps-challenges
  - mobile
  - open-data
```

Reply:
Just remove:

  - citizen-science-and-crowdsourcing-community-of-practice
  - crowdsourcing
  - FCPCCS
  - nasa
  - trends-on-tuesday

Also left comment in 60-day spreadsheet about the `trends-on-tuesday` topic.